### PR TITLE
Enhancement to add ECMAScript 6 support (#1574)

### PR DESCRIPTION
**Pull request by speckyspooky**
Enhancement to add ECMAScript 6 support (#1574)

### CODE REVIEW

**Comment by speckyspooky** [comment]
**Added option of ECMAScript 6 and configuration**

JavascriptVersion.java
	- content:	initialization of the JavaScript engine
	- change:	new class to analysis the configured JVM parameter to get the JavaScript language version number
	- default:	ES6

JavascriptEngine.java
	- content:	initialization of the JavaScript engine
	- change:	set JavaScript language version on the JavaScript engine

ScriptValidator.java
	- content:	validator of the JavaScript scripts
	- change:	set JavaScript language version on the JavaScript engine

MANIFEST.MF of ScriptValidator
	- change:	import the class "JavascriptVersion.java"

**New report version number**

IDesignSchemaConstants.java
	- content:	define current report version nummer
	- change:	changed default report version number "3.2.25"

VersionUtil.java
	- content:	available report version numbers
	- change:	added new version number "3.2.25"

File: data/org.eclipse.birt.report.engine.script.javascript/src/org/eclipse/birt/report/engine/javascript/README.md
Comment (hvbtup) [inline data/org.eclipse.birt.report.engine.script.javascript/src/org/eclipse/birt/report/engine/javascript/README.md:1]: The markdown file contains HTML break elements. Why? I think it's better to just use an empty line to create paragraphs.

**Review by wimjongman — APPROVED** [review]
Woot! Thanks, Thomas!

File: data/org.eclipse.birt.report.engine.script.javascript/src/org/eclipse/birt/report/engine/javascript/README.md
Comment (speckyspooky) [reply data/org.eclipse.birt.report.engine.script.javascript/src/org/eclipse/birt/report/engine/javascript/README.md:1]: Changed.

**Pull request merged**